samples/logging/datadog/sample.md

```diff
@@ -38,7 +38,7 @@ Setup Datadog client. Let' s assume the Datadog agent runs on default address an
 
 snippet: setup-datadog-client
 
-Custom observers need to be registered for the metric probes provided via `NServiceBus.Metrics`. This is configured in the `DatadogFeature`. The registered observers convert NServiceBus.Metric *Signals* to Datadog *Count* and NServiceBus.Metric *Durations* to Datadog *Timer* 
+Custom observers need to be registered for the metric probes provided via the `NServiceBus.Metrics` package. The registered observers convert NServiceBus.Metric *Signals* to Datadog *Count* and NServiceBus.Metric *Durations* to Datadog *Timer*.
```

Comment:
The v7 version of the sample still uses the feature. Is it worth backporting this change (or making the v8 sample still a feature)?

Reply:
I don't see a problem with v7 using a feature, the text doesn't go deeper so I don't think this is a confusing change.

But we can't use a feature unless we take some extra effort to pass everything through the settings to set up the observes in the feature instead. I think we should not use a feature. If we really want, we can encapsulate the setup in a dedicated class but I didn't see that much value in that.

Reply:
Fair enough. The feature isn't the point of the sample and people are unlikely to download both versions and compare them, wondering why they're different.